src/src_cpp/src_lpnamer/LinkdataRecord.cpp

```diff
@@ -0,0 +1,98 @@
+#include "StudyUpdater.h"
+
+
+
+LinkdataRecord::LinkdataRecord(bool modernAntaresVersion_p):
```

Comment:
What is a "modern" version of Antares ? Do we really want to use an older version of Antares ?